libs/libid3tag/Makefile

```diff
@@ -8,61 +8,44 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libid3tag
-PKG_VERSION:=0.15.1b
-PKG_RELEASE:=4
+PKG_VERSION:=0.16.3
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@SF/mad
-PKG_HASH:=63da4f6e7997278f8a3fef4c6a372d342f705051d1eeb6a46a86b03610e26151
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
```

Comment:
Nope.

Reply:
What he said. If upstream doesn't create proper tarballs, switch to OpenWrt ones with PKG_SOURCE_PROTO:=git

Such a thing needs a treewide commit IMO.

Reply:
Switched to git source